packages/backend/src/payment-method/ilp/connector/core/telemetry.ts

```diff
@@ -50,6 +50,7 @@ export async function incrementAmount(
         assetCode: code,
         assetScale: scale
       },
+      undefined,
```

Comment:
If the [sourceAccount](https://github.com/interledger/rafiki/blob/2b428f57761dcd614f926214bff9832164e7d804/packages/backend/src/payment-method/ilp/connector/core/rafiki.ts#L157) in the caller for `incrementAmount` has a known `staticIlpAddress`, it might be possible to acquire a tenantId for this telemetry call from the peer with that address (once tenanted peers are added).